nxdrive/updater/README.md

```diff
@@ -88,16 +88,29 @@ Example of `version.yml` content:
             dmg: ...
             exe: ...
 
+    4.0.0:
+        min_all:
+            '7.10': '7.10-HF44'
+            '8.10': '8.10-HF35'
+            '9.10': '9.10-HF15'
+        type: beta
+        checksum:
+            algo: MD5
+            dmg: ...
+            exe: ...
+
 Each entry describes a version with:
 - `type`: the release type, either an `archive`, a `beta` or a `release`. **Mandatory**.
 - `checksum`: list of checksums for related files. **Mandatory**.
     - `algo`: the algorithm used, it must be one of the [hashlib](https://docs.python.org/2/library/hashlib.html) module (will use SHA256 by default).
     - `dmg`: the checksum of the file `.dmg`. **Mandatory** if you provide a macOS installer.
     - `exe`: the checksum of the file `.exe`. **Mandatory** if you provide a Windows installer.
-- `min`: the minimum Nuxeo version required for this release to work with. **Mandatory**.
-- `max`: the maximum Nuxeo version required for this release to work with. If not defined, Drive will consider the current Nuxeo version as acceptable to work with.
+- `min`: the minimum Nuxeo version required for this release to work with. **Not recommanded, use min_all**.
+- `min_all`: equivalent to `min` but with a finer grain. **Mandatory**.
```

Comment:
Maybe specify that for now this is only informative and not taken into account for the updates yet?